### ISSUE TITLE
 Swedbank Pay logo in header should be an image tag with alt text

### ISSUE THREAD

**Issue by rudgutter** [SwedbankPay/design.swedbankpay.com#44]
[WCAG 2.0 Success Criterion 1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html) requires that all non-text content has an `alt` text. According to the [MDN HTML attribute reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) the only elements that officially support the `alt` attribute are `applet`, `area`, `img` and `input`. As the logo in the header is an `a` tag it's not WCAG compliant. If you give the `a` tag the `alt` attribute it is rendered, but we cannot expect screen readers to pick this up as it's not officially supported.

The logo in the header should be changed to be an `a` tag containing an `img` tag with an `alt` text.

**Comment by Akswii** [comment]
Agreed, will fix.

**Comment by Akswii** [comment]
Closed by #54
